modules/options.rb

```diff
@@ -0,0 +1,11 @@
+module Options
+  def options
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
```

Comment:
Good job refactoring your code using first of all the YAGNI, DRY, KISS principles, then the SOLID principle 👏. When you refactor your code it is supposed to work properly. I noticed that when the user wants to create rentals it's showing the error below, Kindly fix that to go to the next step 😄 

![Capture](https://github.com/ernestmusong/School-library/assets/80458199/cb45cdf5-5fea-401c-bee1-b5ddd6e11304)

Kindly check [this friendly link](https://stackoverflow.com/questions/21821678/ruby-undefined-method-for-nilnilclass-nomethoderror) for details about the bug 😄